features/admin/admin_journey.feature

```diff
@@ -194,6 +194,22 @@ Scenario: As an admin user I want to view Service updates
   When I click 'Service updates'
   Then I am presented with the Service updates page
 
+Scenario: As a normal admin user I should not be able to edit a supplier declaration. Link should not exist on the services page.(This is only available to a CCS admin user)
```

Comment:
`(This is only available to a CCS admin user)` - we have two types of CCS admin user - "category" and "sourcing", but this function is only available to one of them (sourcing I think?) - should we make the exact role explicit in the scenario description?